core/standalone/src/main/scala/org/apache/openwhisk/standalone/StandaloneOpenWhisk.scala

```diff
@@ -393,6 +417,42 @@ object StandaloneOpenWhisk extends SLF4JLogging {
     Await.result(g, 5.minutes)
   }
 
+  private def startKafka(workDir: File, dockerClient: StandaloneDockerClient, conf: Conf, kafkaUi: Boolean)(
+    implicit logging: Logging,
+    as: ActorSystem,
+    ec: ExecutionContext,
+    materializer: ActorMaterializer): (Int, Seq[ServiceContainer]) = {
+    val kafkaPort = getPort(conf.kafkaPort.toOption, 9092)
```

Comment:
If we add a default value to the `kafkaPort` option, can probably solve this differently and not repeat the port number literal here?

Reply:
Ack. Done as you suggested by moving defaults to constant and reusing that across various places